lib/msf/core/module_manager/loading.rb

```diff
@@ -116,7 +116,9 @@ def load_modules(path, options={})
 
     loaders.each do |loader|
       if loader.loadable?(path)
-        count_by_type = loader.load_modules(path, options)
+        count_by_type.merge!(loader.load_modules(path, options)) do |key, old, new|
+          old + new
```

Comment:
No. Github's highlighter is overzealous in making `Klass.new` look special and highlights this case. If we were to invoke `new` here, we would need to to `self.class.new`. Though I can change it if you think it will cause confusion.